eslint-remote-tester.config.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+/** @type {import('eslint-remote-tester').Config} */
+module.exports = {
+  /** Repositories to scan */
+  repositories: [
+    // A few dozen top ESLint plugins.
+    'Intellicode/eslint-plugin-react-native',
+    'JoshuaKGoldberg/eslint-plugin-expect-type',
+    'SonarSource/eslint-plugin-sonarjs',
+    'avajs/eslint-plugin-ava',
+    'cypress-io/eslint-plugin-cypress',
+    'dangreenisrael/eslint-plugin-jest-formatting',
+    'ember-cli/eslint-plugin-ember',
+    'emberjs/eslint-plugin-ember-internal',
+    'eslint-community/eslint-plugin-eslint-plugin',
+    'eslint-community/eslint-plugin-n',
+    'eslint-community/eslint-plugin-promise',
+    'eslint-community/eslint-plugin-security',
+    'eslint-functional/eslint-plugin-functional',
+    'eslint/eslint',
+    'import-js/eslint-plugin-import',
+    'jest-community/eslint-plugin-jest',
+    'jest-community/eslint-plugin-jest-extended',
+    'jsx-eslint/eslint-plugin-jsx-a11y',
+    'jsx-eslint/eslint-plugin-react',
+    'lo1tuma/eslint-plugin-mocha',
+    'ota-meshi/eslint-plugin-regexp',
+    'platinumazure/eslint-plugin-qunit',
+    'sindresorhus/eslint-plugin-unicorn',
+    'square/eslint-plugin-square',
+    'storybookjs/eslint-plugin-storybook',
+    'testing-library/eslint-plugin-jest-dom',
+    'testing-library/eslint-plugin-testing-library',
+    'typescript-eslint/typescript-eslint',
+  ],
+
+  /** Extensions of files under scanning */
+  extensions: ['js', 'mjs', 'cjs', 'ts', 'mts', 'cts'],
+
+  /** Optional boolean flag used to enable caching of cloned repositories. For CIs it's ideal to disable caching. Defaults to true. */
+  cache: false,
```

Comment:
@AriPerkkio I was a bit confused by this `cache` option. I some some other repos disabling it. Can I just let it use the default? Or why is the default value not sufficient?

Reply:
This flag controls whether a cloned repository should be removed from file system after it was linted. The default value `true` is ideal for local development, where developer may have 100s of repositories cached in file system, and they keep linting against them after modifying rule implementation frequently. Cloning repositories is slow, so caching them makes testing fast. Default values of this tool are good for local development, so some changes may be required when targeting CIs. 

In this case, as there are not that many repositories listed, it might be OK to leave this as `true`. Github CIs have 15-20GB file systems on free tier. When linting ~10K repositories, the caching must be disabled so that we don't run out of disk space after couple of hours.